recipe/meta.yaml

```diff
@@ -8,11 +8,10 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 5fd4d7aefa0896fe4dd32815ead8a52ed5ccb8016c7c5743fe8ff71c3c074120
+  sha256: f9fe0698cc497861bdb54cd16161c804031f758ada9480c35540f20c0c078385
 
 build:
   number: 0
-  noarch: python
```

Comment:
Skip py<37, see https://github.com/aio-libs/aiobotocore/blob/fbfeb1242ab1f378f6d4fe84b5d3dacdd8874189/setup.py#L65

Reply:
This has been added